data.js

```diff
@@ -1,4 +1,21 @@
 module.exports = [
+   {
+    name: "Manish Prasad",
+    description:
+      "Founding Team Member | 👨🏻‍💻 Developer | ✍️ Blogger",
+    url: "https://techinsights.manisuec.com",
+    twitter: "@_manish25",
+    tags: [
+      "React",
+      "Nodejs",
+      "Nodej.js",
```

Comment:
Is that `Node.js`?

Reply:
yes, since both are treated as different tags

Reply:
I mean there is a typo. It's not `Nodej.js`, should be `Node.js`

Reply:
updated